static/scss/pages/settings.scss

```
@@ -0,0 +1,125 @@
+.settings-main {
+
+    .c-settings-headline {
+        @include text-title-2xs;
+        padding: $spacing-lg $spacing-lg $spacing-md;
+        color: $color-grey-60;
+    }
+
+    .c-meta--links {
+        display: none;
+    }
+
+    .c-settings-form--heading {
+        @include text-title-3xs;
+        color: $color-grey-50;
+        font-weight: normal;
```

nit: We have been using numbered units to set font weighs else where. 
```suggestion
        font-weight: 400;
```

---

https://github.com/mozilla/fx-private-relay/pull/1180/commits/0afc499f686fa6c1775ee38d4ef93cdab34bfc4d